app/errors.py

```diff
@@ -0,0 +1,22 @@
+class NotWearingMaskError(Exception):
```

Comment:
Consider providing a default error message for your exception. For example: `class NotWearingMaskError(Exception): def __init__(self, message='The user is not wearing a mask'): super().__init__(message)`